common-lib/src/main/scala/com/gu/mediaservice/lib/argo/ArgoHelpers.scala

```diff
@@ -54,6 +55,7 @@ trait ArgoHelpers extends Results {
 
   // TODO: find a nicer way to serialise ErrorResponse[Nothing] without this hack
   def respondError(errorStatus: Status, errorKey: String, errorMessage: String, links: List[Link] = Nil): Result = {
+    Logger.warn(s"Responding with error status $errorStatus, $errorMessage")
```

Comment:
Looks like it needs to `${errorStatus.header.status}` - will double check on TEST it's doing the right thing.